config/rules/no-css-modules.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+module.exports = {
+  create: function (context) {
+    return {
+      ImportDeclaration: function(node) {
+        if (node) {
```

Comment:
Its guarded in [one of the eslint rules](https://github.com/eslint/eslint/blob/master/lib/rules/no-restricted-imports.js#L38) with `ImportDeclaration`, happy to remove it if you dont think its necessary